### ISSUE TITLE
Problems with single column

### ISSUE THREAD

**Issue by foaxdev** [sdras/cssgridgenerator#87]
When set column quantity to 1 (+ 5 rows by default). The grid shows on preview 1 row with 5 columns, instead of 1 column and 5 rows. The code in the "Your Grid Code" is correct.
In Dev Tools  you can see  `grid-template-columns: repeat(5, 1fr);`, must be  `grid-template-columns: 1fr; `

**Comment by kavisherlock** [comment]
Are you sure? Unless it was recently fixed (the last commit to master was over a month ago), I don't see what you're describing. Maybe you're looking at an older version?
I see one column with five rows, and the html for the grid section element has style attribute = "grid-template-columns: 1fr; grid-template-rows: repeat(5, 1fr); gap: 0px;"

![image](https://user-images.githubusercontent.com/23285466/69011895-2c566e80-0935-11ea-8bf9-98000c39c301.png)

**Comment by foaxdev** [comment]
This issue appears in Safari only. Now I've checked in Firefox and everything works perfectly. 
Safari Version 13.0.3 (13608.3.10.10.1)

**Comment by kavisherlock** [comment]
Yup. I see it now. It is very interesting. Only happens when I select 5 and replace it with a 1. If I backspace and then type 1 it's fine. I wonder if it has something to do with VueX incorrectly updating the store in Safari. Weird that it only happens with 1 though. Will have to investigate more
